internal/common/backoff/retrypolicy.go

```diff
@@ -28,25 +28,19 @@ import (
 	"math"
 	"math/rand"
 	"time"
+
+	"go.temporal.io/sdk/internal/common/retry"
 )
 
 const (
-	// NoInterval represents Maximim interval
-	NoInterval                      = 0
-	done              time.Duration = -1
-	noMaximumAttempts               = 0
-
-	// DefaultBackoffCoefficient is default backOffCoefficient for retryPolicy
-	DefaultBackoffCoefficient = 2.0
-	defaultMaximumInterval    = 10 * time.Second
-	defaultExpirationInterval = time.Minute
-	defaultMaximumAttempts    = noMaximumAttempts
+	done time.Duration = -1
 )
 
 type (
 	// RetryPolicy is the API which needs to be implemented by various retry policy implementations
 	RetryPolicy interface {
-		ComputeNextDelay(elapsedTime time.Duration, numAttempts int) time.Duration
+		ComputeNextDelay(elapsedTime time.Duration, attempt int) time.Duration
+		IntoGrpcRetryConfig() *retry.GrpcRetryConfig
```

Comment:
GrpcRetryConfig()

Reply:
I see Rust naming schemes are getting to you :smile:  -- this would be `AsGrpcRetryConfig` in Rust-speak anyway, since the `RetryPolicy` struct isn't destroyed in the process. That name would be reasonable too -- but I agree the `Into` is a little confusing here.

Reply:
Yep, ideally you want just plain `Into`, but unfortunately go can't do that generically. Renamed to `GrpcRetryConfig`.